### PR TITLE
fioctl/git/Troubleshooting: add tip to provide helpful info to Troubleshooting the gitconfig

### PR DESCRIPTION
**Pull request by camilamacedo86**
Signed-off-by: Camila Macedo <camila.macedo@foundries.io>

## Readiness

* [x ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

### CODE REVIEW

**Comment by doanac** [comment]
Docs for e494c7c67dc956b5a38e41746e00bb8d7be4fd32 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2561/docs/artifacts/html/index.html

File: source/user-guide/troubleshooting/troubleshooting.rst [hunk 1]
```diff
@@ -51,6 +51,16 @@ taken to get the exact same result:
...
+.. tip::
+
+    For troubleshooting, prepend ``GIT_CURL_VERBOSE=1 GIT_TRACE=1`` to your ``git clone``
+    command. This modification will provide more detailed information during
+    the cloning process. Carefully review the logs to ensure that the ``git-credential-fio``
+    helper has been executed for credential management.
+
+    To verify that ``git-credential-fio get`` is functioning correctly, run this command
+    and then press ``Control + D``. This action allows you to check if the password
+    required for logging into ``source.foundries.io``.
```
Comment (kprosise) [inline source/user-guide/troubleshooting/troubleshooting.rst:63]:
```suggestion
    required for logging into ``source.foundries.io`` is correct.
```
I am assuming that is what is being checked?

**Review by kprosise — APPROVED** [review]
Just one point to address, otherwise it LGTM!

**Comment by doanac** [comment]
Docs for 65f036b3669babe4128f75b56806687570bc99d8 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2563/docs/artifacts/html/index.html

**Pull request merged**